tests/rpc/personal_test.go

```diff
@@ -142,3 +143,13 @@ func TestPersonal_LockAccount(t *testing.T) {
 	_, err = CallWithError("personal_sign", []interface{}{hexutil.Bytes{0x88}, addr, ""})
 	require.Error(t, err)
 }
+
+func TestPersonal_Unpair(t *testing.T) {
+	t.Skip("skipping TestPersonal_Unpair")
+
+	rpcRes := Call(t, "personal_unpair", []interface{}{"", 0})
+
+	var res error
+	err := json.Unmarshal(rpcRes.Result, &res)
+	require.Equal(t, fmt.Errorf("smartcard wallet not supported yet"), err)
```

Comment:
use `errors.Is`

Reply:
thanks for your advice and correction.
I will add the changelog entry and make the correction.

Reply:
changelog entry added and correct to use errors.Is